packages/services/src/types/index.ts

```diff
@@ -34,7 +34,16 @@ export type SpecificChainBalances = {
   [K in SpecificChain]?: Record<string, number>;
 };
 
-export type SpecificChainTokens = typeof specificChainTokens;
+type TokenDict = Record<string, string>;
+type KnownChains = keyof typeof specificChainTokens;
+
+/**
+ * Standard chain tokens with optional additional tokens
+ */
+export type SpecificChainTokens = Partial<{
```

Comment:
we had some existing tests that try to add new token keys or new chains. that variable is derived from this object:
```ts
declare const specificChainTokens: {
    readonly eth: {
        readonly eth: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
        readonly usdc: "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
        readonly usdt: "0xdAC17F958D2ee523a2206206994597C13D831ec7";
    };
...
```

with this change, it just makes it more flexible so that you can add chains/tokens but still access the default list